deeptime/markov/msm/markov_state_model.py

```diff
@@ -1320,3 +1305,31 @@ def select(self, model_index):
                          reversible=self.reversible, count_model=self._count_models[model_index],
                          transition_matrix_tolerance=self.transition_matrix_tolerance)
         self._current_model = model_index
+
+
+def _svd_sym_koopman(msm: MarkovStateModel, empirical: bool) -> CovarianceKoopmanModel:
+    """ Computes the SVD of the symmetrized Koopman operator in the empirical distribution, returns as Koopman model.
+    """
+    K = msm.transition_matrix
+    if empirical:
+        C = msm.count_model.count_matrix
+        if issparse(C):
+            C = C.toarray()
+        cov0t = C
+        cov00 = np.diag(cov0t.sum(axis=1))
+        covtt = np.diag(cov0t.sum(axis=0))
+        cov = CovarianceModel(cov_00=cov00, cov_0t=cov0t, cov_tt=covtt)
+        # reweight operator to empirical distribution
+        C0t_re = cov00 @ K
```

Comment:
I am bit lost and curious about the goal here. So as far as I understand you estimate here a count matrix which you should have observed given the transition matrix K and the starting points in cov00. But in which sense does this make K symmetric in the next step?